azure/basic-arm-template/README.md

```diff
@@ -0,0 +1,95 @@
+# Basic Arm Template
+
+## Introduction
+
+Azure Resource Manager is the deployment and management service for Azure. It provides a consistent management layer that enables you to create, update, and delete resources in your Azure subscription. You can use its access control, auditing, and tagging features to secure and organize your resources after deployment.
+
+Use this blueprint to provision a single VM using an ARM template.
+
+## Before you get started
+
+If you're new to XebiaLabs blueprints, check out:
+
+* [Get started with DevOps as Code](https://docs.xebialabs.com/xl-platform/concept/get-started-with-devops-as-code.html)
```

Comment:
This URL and the one in the next line doesn't work. I think now is not under `xl-platform` but `xl-release`

Reply:
Fixed both URLs